sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -860,12 +882,148 @@ def locked_until_utc(self):
 
 class AMQPAnnotatedMessage(object):
     """
-    The internal AMQP message that this ServiceBusMessage represents.  Is read-only.
+    The AMQP Annotated Message for advanced sending and receiving scenarios which allows you to
+    access to low-level AMQP message sections.
+    Please refer to the AMQP spec:
+    http://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-messaging-v1.0-os.html#section-message-format
+    for more information on the message format.
+
+    :keyword data_body: The body consists of one or more data sections and each section contains opaque binary data.
+    :paramtype data_body: Union[str, bytes, List[Union[str, bytes]]]
+    :keyword sequence_body: The body consists of one or more sequence sections and
+     each section contains an arbitrary number of structured data elements.
+    :paramtype sequence_body: Union[List[Any], List[List[Any]]]]
```

Comment:
Technically the 2nd of these types is already included in the first.

Reply:
updated :)